# jaguar arm motor command message from DrRobot Robot.
float32 joint